.clang-format

```diff
@@ -1,7 +1,26 @@
 BasedOnStyle:  LLVM
 ---
 AlwaysBreakTemplateDeclarations: true
-BreakBeforeBraces: Allman
+BreakBeforeBraces: Custom
+BraceWrapping:
+  AfterCaseLabel: true
+  AfterClass: true
+  AfterControlStatement: Always
+  AfterEnum: true
+  AfterFunction: true
+  AfterNamespace: true
+  AfterObjCDeclaration: true
+  AfterStruct: true
+  AfterUnion: true
+  AfterExternBlock: true
+  BeforeCatch: true
+  BeforeElse: true
+  BeforeLambdaBody: false
+  BeforeWhile: true
+  IndentBraces: false
+  SplitEmptyFunction: true
+  SplitEmptyRecord: true
+  SplitEmptyNamespace: true
```

Comment:
Do we need to provide all of the values, or just the changed ones?

Reply:
You need all of them. We can't just say "use this style; but change those values". I don't know what the default for `Custom` is, so I would provide everything.